### PR TITLE
Refactor to use es6 features

### PR DESCRIPTION
**Pull request by cloudhary**

### CODE REVIEW

**Comment by cloudhary** [comment]
Sorry, opened it prematurely. 

As per your comment on #77, I've used ES6 features (specifically: classes, Object.assign, let/const). This PR also includes a minor refactor of code with the creation of a `expiration` function. Dropped support for node 5 and added node 11 as per travis.yml. 

I apologize for the one big commit. I'll change it if it's an annoyance

**Comment by mweibel** [comment]
Thanks for the PR! Will look at it soon.

File: lib/connect-session-sequelize.js [hunk 1]
```diff
@@ -5,170 +5,181 @@
...
     }
 
-    var defaults = { 'data': stringData, 'expires': expires }
-    if (this.options.extendDefaultFields) {
-      defaults = this.options.extendDefaultFields(defaults, data)
-    }
+    set (sid, data, fn) {
+      debug('INSERT "%s"', sid)
+      const stringData = JSON.stringify(data)
+      let expires = this.expiration(data)
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: I believe this can be a `const`

File: lib/connect-session-sequelize.js [hunk 2]
```diff
@@ -5,170 +5,181 @@
...
-      debug('TOUCH skipped due to disableTouch "%s"', sid)
-      return fn()
-    }
+      if (this.options.disableTouch) {
+        debug('TOUCH skipped due to disableTouch "%s"', sid)
+        return fn()
+      }
 
-    var expires
+      let expires = this.expiration(data)
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: can be a `const` too, I think

File: lib/connect-session-sequelize.js [hunk 3]
```diff
@@ -5,170 +5,181 @@
...
 
-  SequelizeStore.prototype.stopExpiringSessions = function stopExpiringSessions () {
-    if (this._expirationInterval) {
-      clearInterval(this._expirationInterval)
-      // added as a sanity check for testing
-      this._expirationInterval = null
+    expiration (data) {
+      if (data.cookie && data.cookie.expires) {
+        return data.cookie.expires
+      } else {
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: else is not strictly needed as you return anyway

**Comment by mweibel** [comment]
Also refactoring the expiration to an own function: 👍 nice.

**Comment by cloudhary** [comment]
I'm sorry I never finished this! Give me some time and I'll get to it. Thanks for your patience @mweibel!

**Comment by cloudhary** [comment]
I've made the changes as proposed. Feel free to merge it in if you think it's good, and let me know if you'd like me to make any other amends. 

Thank you for your patience with me @mweibel!

**Comment by mweibel** [comment]
👍 thanks! Will release soon.

**Pull request merged**